src/v/cluster/members_backend.cc

```diff
@@ -294,6 +294,18 @@ void members_backend::calculate_reallocations_after_node_added(
         return lhs.left_to_move < rhs.left_to_move;
     };
 
+    if (clusterlog.is_enabled(ss::log_level::info)) {
```

Comment:
interesting. i assumed our default log level was info. i guess maybe run at warn level too?

Reply:
yes, warn level is possible